package.json

```diff
@@ -27,7 +28,10 @@
     "tar": "^1.0.3"
   },
   "config": {
-    "excludeAbi": "11"
+    "excludeAbi": "11",
+    "oracleUrl": "http://7e9918db41dd01dbf98e-ec15952f71452bc0809d79c86f5751b6.r22.cf1.rackcdn.com",
+    "oracleVersion": "2.0.0",
+    "driverModule": "oracledb@^1.11.0"
```

Comment:
So, reverting back to what was before..  
 "config": {
    "excludeAbi": "11"
  } remains in oracle installer as before and rest of the config will be reverted back to oracle connector?